src/app/GitCommands/Settings/AppSettings.cs

```diff
@@ -1523,9 +1523,9 @@ public static bool FillRefLabels
 
         public static ISetting<bool> MergeGraphLanesHavingCommonParent { get; } = Setting.Create(RevisionGraphSettingsPath, nameof(MergeGraphLanesHavingCommonParent), true);
 
-        public static ISetting<bool> RenderGraphWithDiagonals { get; } = Setting.Create(ExperimentalSettingsPath, nameof(RenderGraphWithDiagonals), true);
+        public static ISetting<bool> RenderGraphWithDiagonals { get; } = Setting.Create(RevisionGraphSettingsPath, nameof(RenderGraphWithDiagonals), true);
 
-        public static ISetting<bool> StraightenGraphDiagonals { get; } = Setting.Create(ExperimentalSettingsPath, nameof(StraightenGraphDiagonals), true);
+        public static ISetting<bool> StraightenGraphDiagonals { get; } = Setting.Create(RevisionGraphSettingsPath, nameof(StraightenGraphDiagonals), true);
```

Comment:
❓ Change default value of both to `false`? Vote, please!

Reply:
I prefer to have them set (but do not find a good example for StraightenGraphDiagonals now...)

Reply:
> ❓ Change default value of both to `false`? Vote, please!

What do these do?

Reply:
These settings can be changed here:

![image](https://github.com/gitextensions/gitextensions/assets/36601201/c835beda-4278-48a1-a295-748fabe0c7f1)

They have been added with #10637.

Reply:
Works for me